src/types/form.ts

```diff
@@ -301,6 +301,10 @@ export interface IFormModel extends Model<IFormSchema> {
     formId: string,
     fields?: (keyof IPopulatedForm)[],
   ): Promise<IPopulatedForm | null>
+  createFormLogic(
+    formId: string,
+    createdLogic: LogicDto,
```

Comment:
if fixing the nit, also remember to rename it here

Reply:
edited